### DISCUSSION TITLE
is anyone else missing security and network proxy settings?

### DISCUSSION THREAD

**Discussion by aijunction** [Kong/insomnia#7006] [Help]
<img width="779" alt="Screenshot 2024-01-19 at 1 56 51 PM" src="https://github.com/Kong/insomnia/assets/145908416/f5cd6c39-2a1c-4861-8ac7-b25bdb0750bf">

**Comment by subnetmarco** [comment]
In Insomnia 9.x we made it easier to find the Proxy settings, this issue should not happen again.

![Screenshot 2024-07-07 at 10 14 06 PM](https://github.com/Kong/insomnia/assets/964813/be694cc0-317d-4946-a399-0121658b9473)
